__tests__/app.test.js

```diff
@@ -103,7 +103,40 @@ describe('/api/articles', () => {
             expect(body.articles.length).toBe(5)
             const articles = body.articles
             expect(articles).toBeSorted({descending: true})
+            articles.forEach((article)=>{
+                expect(article).toMatchObject({
+                    article_id: expect.any(Number),
+                    title:expect.any(String),
+                    topic:expect.any(String),
+                    author:expect.any(String),
+                    created_at:expect.any(String),
+                    votes:expect.any(Number),
+                    article_img_url:expect.any(String),
+                    comment_count:expect.any(String)
+                })
+            })
         })
     });
 })
 
+describe('GET /api/articles/4/comments', ()=>{
+    test('should return the comments object for the specified article with the correct shape', () =>{
+        return request(app)
+        .get('/api/articles/4/comments')
+        .expect(200)
+        .then(({body})=>{
+            expect(body.comments).toEqual([])
```

Comment:
lets think about what we are testing here and describe it properly. (you seem to be testing for comments from an article that exists but doesn't have any comments)